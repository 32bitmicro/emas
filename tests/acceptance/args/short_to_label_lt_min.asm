label:
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop nop nop nop nop nop nop nop
nop nop nop
ujs label
