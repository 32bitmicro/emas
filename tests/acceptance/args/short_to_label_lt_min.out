acceptance/args/short_to_label_lt_min.asm:9:5: Argument value -64 for INT is out of range (-63..63)
